README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
What technologies did you use? you should better describe your project in my opinion.